example/test.d.ts

```diff
@@ -0,0 +1,1013 @@
+declare namespace proto.jspb.test {
```

Comment:
We don't want namespaces in the es6 output. I think we can just remove this and unindent the contents.